### ISSUE TITLE
How can I make this work

### ISSUE THREAD

**Issue by macasas** [xuzn/pikchr-markdown-preview#1]
Hi, How can I make this work?  When I open the Markdown Preview, I get nothing inside the '''pikchr, does it need another extension to help it?  Thanks

**Comment by nklamann** [comment]
No, it should render a valid piece of pikchr-code like the following, but consider #2 
` ``` `pikchr
    arrow right 200% "Markdown" "Source"
    box rad 10px "Markdown" "Formatter" "(markdown.c)" fit
    arrow right 200% "HTML+SVG" "Output"
    arrow <-> down 70% from last box.s
    box same "Pikchr" "Formatter" "(pikchr.c)" fit

` ``` `

**Comment by macasas** [comment]
that just shows the text as code, i.e with a grey background, as if ignoring the ```pikchr request
the markdown bit is previewing, but the pikchr is doing nothing

**Comment by nklamann** [comment]
Works for me this way . Is the extension enabled ?

**Comment by macasas** [comment]
<img width="1683" alt="Screenshot 2023-01-25 at 14 31 18" src="https://user-images.githubusercontent.com/910082/214590308-eadb9490-34f8-4064-8e41-846b977787c6.png">

Enabled extension, your own example, just previewing as code

**Comment by nklamann** [comment]
Sorry, no idea.

**Comment by macasas** [comment]
For those that follow, my problem was another extension, Markdown Preview Enhanced, which must interfere.  Disabled it, now renders.
